src/components/transactions/delegateStatistics.css

```diff
@@ -0,0 +1,303 @@
+@import '../app/variables.css';
+
+:root {
+  --detail-label-font-weight: var(--font-weight-very-bold);
+  --list-line-height: 25px;
+  --details-label-font-size-XL: 15px;
+  --details-label-font-size-L: 12px;
+  --details-label-font-size-XS: 12px;
+}
+
+.addressLink {
+  color: var(--color-link);
+  font-weight: 600;
+  text-decoration: none;
+
+  &::after {
+    content: "• ";
+    padding: 0px 6px;
+    color: var(--color-primary-standard) !important;
+  }
+
+  &:last-child::after {
+    content: "";
+  }
+}
+
+.filters {
+  font-weight: var(--font-weight-bold);
+  color: var(--color-grayscale-medium);
+  line-height: var(--list-line-height);
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+
+  & .filter {
+    display: inline-block;
+    cursor: pointer;
+    font-size: 15px;
+    font-weight: 500;
+    vertical-align: middle;
+    margin-right: 10px;
+
+    &:not(:last-of-type) {
+      margin-right: 24px;
+    }
+  }
+
+  & .active {
+    border-bottom: 2px solid var(--color-grayscale-dark);
+    color: var(--color-grayscale-dark);
+  }
+}
+
+.search {
+  position: relative;
+  padding-bottom: 10px;
+  color: var(--color-grayscale-medium);
+
+  & input {
+    height: 25px;
+    border: none;
+    outline: none;
+    margin: 0;
+    min-width: 200px;
+    font-size: var(--details-label-font-size-L);
+    font-weight: 600;
+    padding-left: 30px;
+    padding-right: 20px;
+    vertical-align: top;
+    background-color: transparent;
+    border-radius: 0;
+    border-bottom: solid 2px transparent;
+    transition: all ease 250ms;
+
+    &:-moz-placeholder,
+    &::-moz-placeholder,
+    &::-webkit-input-placeholder {
+      font-weight: 400;
+      color: var(--color-grayscale-medium);
+      font-size: 15px;
+    }
+
+    &.desktopInput {
+      display: inline-block;
+    }
+
+    &.mobileInput {
+      display: none;
+    }
+
+    &:focus,
+    &.dirty {
+      border-bottom-color: var(--color-grayscale-medium);
+
+      & ~ .clean {
+        visibility: visible;
+      }
+    }
+  }
+
+  & .clean,
```

Comment:
Why are the styles implemented, but no actual icon for this? When I search, I currently can't clear my search